test/skimNtuple_HHbtag.cpp

```diff
@@ -5271,11 +5318,31 @@ int main (int argc, char** argv)
 		  theSmallTree.m_HHbregrsvfit_phi = tlv_HHbregrsvfit.Phi();
 		  theSmallTree.m_HHbregrsvfit_m	  = tlv_HHbregrsvfit.M();
 
+		  // verify if the fatjet matches any previously found Higgs or Z decaying into bb
+		  bool matchedToHiggs = false;
+		  bool matchedToZ = false;
+		  for (const auto& higgs : higgsVectors) {
```

Comment:
here the higgs is only the b quark, right? Couldn't it happen that somehow the bbar can be matched but not the b?

Reply:
Sorry, I didn't quite understand...